src/core/server/http/lifecycle/on_pre_response.ts

```diff
@@ -55,6 +62,12 @@ export interface OnPreResponseInfo {
 }
 
 const preResponseResult = {
+  render(body: string, statusCode?: number | undefined): OnPreResponseResult {
```

Comment:
Could you add functional test for `render` https://github.com/elastic/kibana/blob/0ea414c13a458d521b5ac9f3b181e12396837009/src/core/server/http/integration_tests/lifecycle.test.ts ?

Reply:
Fixed 👍